installation-and-configuration/office-365-connection-details.md

```diff
@@ -40,6 +40,22 @@ SysKit Trace uses certificates for two reasons.
 A certificate is created and imported into the current user personal certificates for both the user configuring SysKit Trace and the service account that is configured to run the SysKit Trace service.  
 We advise that the machine where SysKit Trace is installed is secure and not shared with other people.
 
+
+### Deleting SysKit Trace Azure AD Application from Azure
+
+Although removing a connection in SysKit Trace configuration wizard will remove all the data related to that connection from your local machine, it will not delete the created Azure AD Application.
```

Comment:
_will remove all the data related to that connection_
does it though? there are still snapshots on the disk :).

Reply:
Should I maybe add _(except created snapshots)_ after _data_ then?